sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/TextDocumentInput.java

```diff
@@ -37,20 +34,9 @@
      * @param text The document to process.
      */
     public TextDocumentInput(String id, String text) {
-        this(id, text, null);
-    }
-
-    /**
-     * Creates a {@link TextDocumentInput} model that describes the documents.
-     *
-     * @param id A unique, non-empty document identifier.
-     * @param text The document to process.
-     * @param language Optional. This is the 2 letter ISO 639-1 representation of a language.
-     */
-    public TextDocumentInput(String id, String text, String language) {
         this.id = id;
         this.text = text;
-        this.language = language;
+        this.language = "en";
```

Comment:
why is the language just set to english? it seems that no matter what you pass in for language, your language is just going to be english